src/server.js

```diff
@@ -53,9 +54,14 @@ module.exports = function() {
 			log.error("Path to SSL certificate is invalid. Stopping server...");
 			process.exit();
 		}
+		if (config.https.ca.length && !fs.existsSync(caPath)) {
```

Comment:
Why? Idea is to check if configuration parameter is wrongly set. That means parameter exists and file is missing.